src/20231106_AaveV3Ethereum_AaveFundingUpdates/AaveFundingUpdates.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Aave Funding Updates (part 2)"
+author: "TokenLogic, efecarranza.eth"
+discussions: "https://governance.aave.com/t/arfc-aave-funding-update/15194"
+---
+
+## Simple Summary
+
+This AIP aims to consolidate Aave DAO's stablecoin holdings. It is divided into different parts that are detailed below. This is part two and it consists of depositing USDC into Aave v3 Ethereum, as well as migrating the v2 position to v3 in Ethereum. Also depositing USDT into Aave v2 Ethereum and swapping DAI for GHO.
```

Comment:
Minor edit, see below comments for context.

...swapping DAI and USDC for GHO.